composer.json

```
@@ -6,10 +6,10 @@
     "license": "BSD-3-Clause",
     "require": {
         "silverstripe/recipe-plugin": "^1.2",
-        "silverstripe/assets": "1.x-dev",
-        "silverstripe/config": "1.0.x-dev",
-        "silverstripe/framework": "4.x-dev",
-        "silverstripe/mimevalidator": "2.x-dev"
+        "silverstripe/assets": "1.6.x-dev",
+        "silverstripe/config": "1.1.x-dev",
```

Config doesn't change that often, so we don't do minor release of it. It pretty much has state the same since the 4.0 release.

```suggestion
        "silverstripe/config": "1.0.x-dev",
```